pkg/util/kubernetes/kubelet/kubelet.go

```diff
@@ -763,7 +774,13 @@ func checkKubeletHTTPSConnection(ku *KubeUtil, httpsPort int) error {
 			return err
 
 		} else if strings.Contains(err.Error(), "x509: certificate signed by unknown authority") {
-			log.Debugf(`The kubelet server certificate is signed by unknown authority, the current cacert is %s. Is the kubelet issuing self-signed certificates? Please validate the kubelet certificate with "openssl verify -CAfile %s ${KUBELET_CERTIFICATE}" to avoid this error: %v`, ku.rawConnectionInfo["ca_cert"], ku.rawConnectionInfo["ca_cert"], err)
+			ku.RLock()
+			certString, ok := ku.rawConnectionInfo["ca_cert"]
+			ku.RUnlock()
+			if !ok {
+				log.Debugf(`The kubelet server certificate is signed by unknown authority, the cacert is not set`)
```

Comment:
if `!ok` you'll see both logs, worth returning here ?

Reply:
I return here, I need to update the Debug message.

I will do it